### ISSUE TITLE
Time span of subscribed harvests

### ISSUE THREAD

**Issue by lbross** [LANDIS-II-Foundation/Library-Harvest-Mgmt#2]
Reported by AMGdeBruijn, Feb 9, 2012
What steps will reproduce the problem?
1. Setting a simulation length shorter than time span in harvest files

What is the expected output? 
Model should run

What do you see instead?
Error: in harvest file: "year x is after the scenario' end year <y>" 

What version of the product are you using? On what operating system?
6.0/windows 7

Please provide any additional information below.

Having a simulation length that does not cover harvest presciptions 
should not be an error because it can be a desired model setup to run 
only a section of the harvests program that is defined in the harvest 
file. Alternatively consider supporting multiple lines comments to 
allow users to reduce simulation time and to comment out a fraction of 
the prescribed harvests w/o having to outcomment individual lines in 
harvest file. 

Jul 25, 2013 Project Member #1 rmscheller
We could change this from an error to a note:  "Note: year <x> is after the scenario' end year <y> on line <z> of the harvest input file."

Apr 16, 2014  #2 AMGdeBruijn
That would work

Nov 7, 2014 Project Member #3 lbross@pdx.edu
Issue assigned to Lesley Bross by Brian/Eric. Could either Brian or Arjan provide a sample scenario file that shows this bug? The simpler, the better.

Nov 7, 2014 #4 AMGdeBruijn
This will run with the example simulation that comes with the base harvest installer.

It's not really a bug, just a nuisance; wen I want to run a couple of timesteps of a complex simulation, I now have to outcomment lines in the harvest file.

---

LandisData  "Biomass Harvest"

Timestep    10

ManagementAreas "./management.gis"

Stands      "./stand.gis"

> > ---
> > 
> > PRESCRIPTIONS

Prescription    AspenClearcut
    StandRanking    Economic
    >> Species   Economic Rank  Minimum Age
    >> -------   -------------  -----------
       poputrem   100           40    

```
MinimumAge      35
SiteSelection   CompleteStandSpread 40 50
CohortsRemoved  ClearCut
```

> > ---
> > 
> > PRESCRIPTION IMPLEMENTATION 

```
HarvestImplementations
```

> >  Mgmt Area   Prescription            Harvest Area    Begin Time  End Time
> > ---
> > 
> > ```
> > 2   AspenClearcut           5%      1000    5000
> > ```

Nov 7, 2014 Project Member #5 lbross@pdx.edu
Received test cases from Arjan and Brian. Thank you!

Google code issue #4

**Comment by lbross** [comment]
Eric to validate this enhancement because the original requester, Arjan, is too busy

**Comment by ArjandeBruijn** [comment]
I can do it, it prob takes longer to remind Eric what the issue was than for me to try it. 

Where is the new code? I'm still kinda new to github

**Comment by lbross** [comment]
Thanks! We aren't quite ready to validate yet as we first need to make sure that the harvest extensions with the new libraries work the same as the old ones. Following that, we will start looking at enhancements, but I'll let Eric know you offered.

If you want to look at the code, it is at https://github.com/LANDIS-II-Foundation/Extension-Base-Harvest and https://github.com/LANDIS-II-Foundation/Extension-Biomass-Harvest.

**Comment by lbross** [comment]
Fixed in v1.1.0
